src/App.jsx

```diff
@@ -116,9 +116,9 @@ function App() {
       <Switch>
         <Route exact path='/' component={Home} />
         <Route path='/create-event' component={CreateEventPage} />
-        <Route exact path='/events/:id' component={EventView} />
```

Comment:
update form will not properly render at `/events/:id/update` if `/events/:id` is not an exact path